### DISCUSSION TITLE
Valheim Patch 0.209.9

### DISCUSSION THREAD

**Discussion by awallace47** [lloesche/valheim-server-docker#489] [Q&A]
With this new update it looks like the devs have moved some file paths around. Is anyone tracking these updates to support this new version?

**Comment by CygnSoft** [comment]
With the new update, Valheim Plus isn't working any more, because BepInEx needs an Update, but the Updater won't install it. If you disable Valheim Plus and enable BepInEx it's working again, because the Valheim Plus dependency is gone. 
I did the following to fix the Problem manually:

The "mainfolder" would be /opt/valheim/plus, if you're playing with V+
I've created a new folder in there, where I downloaded the newest Version of BepInEx (wget https://gcdn.thunderstore.io/live/repository/packages/denikson-BepInExPack_Valheim-5.4.1901.zip)
I unzipped everything into the new folder, wrote down all files and directories inside, and made a backup of the files and directories in the actual folder. 
Then I replaced the old files and directories with the new ones, restarted the container, and voila, the Server is accessable again.

But there is one big issue left: The new Bepinex Folder doesnt have the config symlinks, so I recreated them and replaced the config folder again with the old one in the Backup folder. But as soon as I join the Server, I don't have the Valheim Plus Config changes in the game, like bigger inventory and stuff. I didn't fixed that problem yet.

@lloesche would it be possible to get an updated Container with the new BepInEx Version?

**Reply by AdamLMark** [reply]
The fix for me was to move my world(s) from /config/worlds to /config/worlds_local (had to overwrite files)

**Reply by kripton** [reply]
You just saved my day! I was totally shocked being greeted by an empty world on Day 1 on my dedicated server. Instead of moving the files, I made a symlink from `worlds_local` to `worlds` in order to keep the auto-backup system working.
